doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -1,3 +1,262 @@
 # Python SDK
 
-More docs coming soon!
+The code for the Python SDK lives in `/econia/src/python/sdk/econia_sdk` ([link](https://github.com/econia-labs/econia/tree/main/src/python/sdk/econia_sdk)). There are 2 primary packages (`econia_sdk.entry` and `econia_sdk.view`) complemented by two secondary imports
+
+# Primary Packages
+## `econia_sdk.entry`
+
+This package contains helpers for the `public entry` (that is, "targetable by transactions") functions possessed by the Econia exchange. Each method's name corresponds to the name of a public entry function in one of the following Econia Move modules:
+
+- `incentives` ([move](https://github.com/econia-labs/econia/blob/main/src/move/econia/sources/incentives.move)) ([python](https://github.com/econia-labs/econia/blob/main/src/python/sdk/econia_sdk/entry/incentives.py))
+- `market` ([move](https://github.com/econia-labs/econia/blob/main/src/move/econia/sources/market.move)) ([python](https://github.com/econia-labs/econia/blob/main/src/python/sdk/econia_sdk/entry/market.py))
+- `registry` ([move](https://github.com/econia-labs/econia/blob/main/src/move/econia/sources/registry.move)) ([python](https://github.com/econia-labs/econia/blob/main/src/python/sdk/econia_sdk/entry/registry.py))
+- `user` ([move](https://github.com/econia-labs/econia/blob/main/src/move/econia/sources/user.move)) ([python](https://github.com/econia-labs/econia/blob/main/src/python/sdk/econia_sdk/entry/user.py))
+
+The corresponding Move code is well-commented and up-to-date, so it's helpful to have handy when working with the Python SDK. If the function you desire happens to not be supported by the Python SDK (or sadly isn't working), it's possible to use `econia_sdk.lib.EconiaClient` to execute the appropriate `public entry` Move function yourself. See the Python code linked above for examples of how an `EntryFunction` instance is created.
+
+## `econia_sdk.view`
+
+This package contains helps for the `#[view]` (that is, "targetable off-chain") functions possessed by the Econia exchange. Each method's name corresponds to the name of a public view function in one of the following Econia Move modules:
```

Comment:
Will you please simplify the first sentence to:

"This package contains helpers for Econia's `[view]` functions."

In the second sentence, change "public view function" to "view function" to avoid confusion about being public in runtime vs off-chain